rosbag2/src/rosbag2/sequential_reader.cpp

```diff
@@ -41,14 +42,18 @@ void
 SequentialReader::open(
   const StorageOptions & storage_options, const ConverterOptions & converter_options)
 {
+  storage_options_ = storage_options;
   storage_ = storage_factory_->open_read_only(storage_options.uri, storage_options.storage_id);
```

Comment:
This should be updated to reflect the new writer changes, where the URI should change from:
`<file_name>`
to
`<directory_name>/<file_name>`